packages/@okta/i18n/src/properties/login.properties

```diff
@@ -822,8 +822,12 @@ oie.okta_verify.signed_nonce.label = Use Okta Verify on this device
 oie.okta_verify.enroll.force.upgrade.title = Update Okta Verify
 oie.authenticator.app.non_fips_compliant_enrollment_device_incompatible = The device used to set up Okta Verify does not meet your organization’s security requirements because it is not FIPS compliant. Contact your administrator for help.
 oie.authenticator.app.non_fips_compliant_enrollment_app_update_required = The Okta Verify version on the device used does not meet your organization’s security requirements. To add your account, update Okta Verify to the latest version, then try again.
-oie.authenticator.app.method.push.verify.enable.biometrics = We received your response, but your organization requires biometrics. On a device that supports biometrics, enable biometrics for your account and try again.
+oie.authenticator.app.method.push.verify.enable.biometrics = Your response was received, but your organization requires biometrics. Make sure you meet the following requirements, then try again:\\r\\n\\u2022 Your device supports biometrics\\r\\n\\u2022 Okta Verify is up-to-date\\r\\n\\u2022 In Okta Verify, biometrics are enabled for your account\\r\\n\\r\\n
```

Comment:
do we need all the line breaks `\n` in the text? Can this be a simpler sentence instead? I don't think we use it, but if we ever do, it might be better to just have a regular message